borg_import/rsnapshots.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+
+
+def get_snapshots(rsnapshot_root):
+    output = []
+    for dirname, dirnames, filenames in os.walk(rsnapshot_root):
+        for subdir in dirnames:
+            output.append(os.path.join(dirname, subdir))
```

Comment:
a usual pattern in python is to say "for thing in things:". so, you would rather say here "for dirname in dirnames" in the inner loop, but then you need to say "for basedir, dirnames, files ..." in the outer loop.